README.md

```diff
@@ -221,8 +221,11 @@ Repository tags will be in the following format:
 
 |Option|Required|Description|
 |------|--------|-----------|
-|`GITHUB_TOKEN`|Required if `GITHUB_USE_SSH` is not specified|[GitHub personal access token.](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)|
-|`GITHUB_USE_SSH`|Required if `GITHUB_TOKEN` is not specified|Set any value to use SSH with [deploy keys](https://docs.github.com/en/developers/overview/managing-deploy-keys#deploy-keys) or your private SSH key. Your system must ready to use the key as publib will not set it up.|
+|`GITHUB_TOKEN`|Required if either `GITHUB_USE_SSH`\\`GIT_USE_SSH` and combination of `GH_ENTERPRISE_TOKEN`\\`GITHUB_ENTERPISE_TOKEN` and `GH_HOST` is not specified|[GitHub personal access token.](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)|
+|`GITHUB_USE_SSH` or `GIT_USE_SSH`|Required if `GITHUB_TOKEN` and combination of `GH_ENTERPRISE_TOKEN`\\`GITHUB_ENTERPISE_TOKEN` and `GH_HOST`is not specified|Set any value to use SSH with [deploy keys](https://docs.github.com/en/developers/overview/managing-deploy-keys#deploy-keys) or your private SSH key. Your system must ready to use the key as publib will not set it up.|
```

Comment:
Let's align on `GIT_USE_SSH` for most of the doc.
```suggestion
|`GIT_USE_SSH`|Required if `GITHUB_TOKEN` and combination of `GH_ENTERPRISE_TOKEN`\\`GITHUB_ENTERPISE_TOKEN` and `GH_HOST`is not specified|Set any value to use SSH with [deploy keys](https://docs.github.com/en/developers/overview/managing-deploy-keys#deploy-keys) or your private SSH key. Your system must ready to use the key as publib will not set it up.|
|`GITHUB_USE_SSH`|Deprecated. Alias for `GIT_USE_SSH`.|
```

Reply:
sounds good to me, committed!